MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+	</tr>
+    
+    <tr>
+    <td>
+            SfSegmentedControlNormalStroke    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of border in segmented control.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+    <tr>
+    <td>
+            SfSegmentedControlNormalTextColor     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of text in segmented control.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalBackground      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of segmented control.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentBackground       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment when it is disabled. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentTextColor        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment text when it is disabled.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionBackground         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment when it is selected.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionTextColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of text when it is selected. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionStroke           
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of border when it is selected.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlHoveredBackground            
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment when it is hovered.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlKeyboardFocusStroke             
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of border when it is keyboard pressed.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalStrokeThickness              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of stroke thickness in segemented control.    
```

Comment:
can we use ? - Thickness of the stroke in the in segmented control.

Reply:
we used Thickness of the border stroke in the segmented control.